src/managers/Koreanbots.ts

```diff
@@ -1,28 +1,22 @@
 import { snowflakeRegex } from "../util/Constants"
 import APIRouter from "../rest/APIRouter"
-import { Bot } from "src/structures/Bot"
+import { Mybot } from "./Mybot"
+import { BotManager } from "./BotManager"
 
-import type { FetchResponse, KoreanbotsOptions, ProxyValidator } from "../structures/core"
-import type { RequestInit } from "node-fetch"
-
-interface BotQuery {
-    bots(id: string): {
-        stats: {
-            post: (options: RequestInit) => Promise<FetchResponse<Bot>>
-        }
-    }
-}
+import type { KoreanbotsOptions, ProxyValidator } from "../structures/core"
 
 class Koreanbots {
     public readonly options!: KoreanbotsOptions
     public readonly api!: ReturnType<typeof APIRouter>
+    public mybot!: Mybot
+    public bots: BotManager
 
     static validator = <T>(): ProxyValidator<T> => ({
         set(obj, prop, value) {
             switch (prop) {
             case "clientID":
                 if (typeof value !== "string") throw new TypeError(`"clientID" 옵션은 문자열이여야 합니다. (받은 타입: ${typeof value})`)
-                if (!snowflakeRegex.test(value)) throw new TypeError("\"clientID\" 옵션은 디스코드의 ID 체계인 Snowflake여야 합니다.")
+                if (snowflakeRegex.test(value)) throw new TypeError("\"clientID\" 옵션은 디스코드의 ID 체계인 Snowflake여야 합니다.")
```

Comment:
wtf

Reply:
This is correct. ``RegExp.prototype.test`` returns true if matches @wonderlandpark